app/src/main/java/io/intrepid/contest/screens/contestcreation/NewContestActivity.java

```diff
@@ -77,6 +77,8 @@ public void setupViewPager(ViewPager viewPager) {
     public boolean onCreateOptionsMenu(Menu menu) {
         menu.clear();
         getMenuInflater().inflate(R.menu.menu_new_contest, menu);
+        MenuItem nextItem = menu.findItem(R.id.action_next);
+        nextItem.setVisible(nextVisible);
```

Comment:
The accetance criteria uses "grey" but the zeplin file suggests that its just disappearing and reappearing. I'll doublecheck with design.